### ISSUE TITLE
Adding an ActiveDirectory Container settings

### ISSUE THREAD

**Issue by cybrouillard** [jakubgarfield/Bonobo-Git-Server#754]
Hi,

Would it be possible to add a parameter to appSettings for setting an AD container string that will be used in each instantiation of PrincipalContext object.

For some reasons, Groups and Users won't be found in my AD without specifing a container.

Thanks !

**Comment by larshg** [comment]
@cybrouillard Yes, if you can make a PR we can review it and properly merge it.
